ransack-mongoid.gemspec

```diff
@@ -1,39 +1,39 @@
 
-lib = File.expand_path("../lib", __FILE__)
+lib = File.expand_path('../lib', __FILE__)
 $LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
-require "ransack/mongoid/version"
+require 'ransack/mongoid/version'
 
 Gem::Specification.new do |spec|
-  spec.name          = "ransack-mongoid"
+  spec.name          = 'ransack-mongoid'
   spec.version       = Ransack::Mongoid::VERSION
-  spec.authors     = ["Ernie Miller", "Ryan Bigg", "Jon Atack","Sean Carroll"]
```

Comment:
I don't think you get to be added as the author of this gem just with a single contribution :sweat_smile:. I'm grateful, and probably the authors of this gem too, for your contribution though.

In fact, I would personally revert all the changes in this file, since you only changing the quoting style, and adding your name and email address.

Reply:
Thanks for reviewing so intensely. Is there anything that is missing or that requires attention for the gem to be ready for 6 ruby 2.7.0 and mongo 7 ?

Reply:
I have no idea, since it doesn't even seem this repo has TravisCI enabled. We can probably ask the current maintainers to add that?

Have you tried the gem in your application?

Reply:
I am currently using this version, hence the PR request.